torch/_inductor/debug.py

```diff
@@ -466,6 +470,6 @@ def handle_tensor(x):
             return x
 
     fake_mode = torch._subclasses.FakeTensorMode(allow_non_fake_inputs=True)
-    with fake_mode, config.patch("save_args", False):
+    with fake_mode, config.patch("save_args", False):  # type: ignore[attr-defined]
```

Comment:
the config module does some dynamic stuff via `install_config_module` which makes this hard to type